test/functional/test_framework/mininode.py

```diff
@@ -221,19 +221,24 @@ def ToHex(obj):
 
 class CAddress(object):
     def __init__(self):
+        self.time = 0
         self.nServices = 1
         self.pchReserved = b"\x00" * 10 + b"\xff" * 2
         self.ip = "0.0.0.0"
         self.port = 0
 
-    def deserialize(self, f):
+    def deserialize(self, f, without_time=False):
```

Comment:
nit: a default of without_foo = False isn't no bad thing for nobody.

with_time=True please :)